tools/python_api/test/test_pl.py

```diff
@@ -0,0 +1,10 @@
+import polars as pl
+from type_aliases import ConnDB
+
+def test_polars_basic(conn_db_readonly: ConnDB) -> None:
+    conn, db = conn_db_readonly
+    df = pl.DataFrame({"col1": [1, 2], "col2": [3, 4]})
+    result = conn.execute("LOAD FROM df RETURN *").get_as_pl() == df
+    assert result['col1'].all()
+    assert result['col2'].all()
+    # since polars just uses arrow as its backend, it's probably not necessary to extensively test this functionality
```

Comment:
I think the file naming of the tests are inconsistent in this directory:

- `test_df.py` tests for converting data to Pandas
- `test_scan_pandas.py` tests for scanning from Pandas
- `test_pl.py` seems to be testing for scanning from Polars, so it would ideally be named `test_scan_polars.py` so that it's placed near the Pandas tests alphabetically.

That being said, we don't seem to be testing the `get_as_pl()` method at all considering that all it is is a wrapper around `get_as_arrow()`. I

So we have two options:

- Replicate the tests that we have for `test_scan_pandas.py` in the Polars case as well?
- Or, if you think that all we're doing in the scan operation is exposing Arrow types like we are for Pandas, then we could just write tests for the wrapper methods to ensure coverage.

Reply:
I think it's a good idea to rename the files.

When scanning from polars DFs, we basically treat the DF as a wrapper on top of an arrow table. Since scanning from arrow tables has already been tested quite well in test_df_pyarrow.py (note to self: rename this as well) I think it's not necessary to do more testing than to just test the wrapper.

Reply:
Yeah I think this is the same logic we used in the `get_as_pl()` too, so I think just testing the wrappers is a good way to go for Polars.